test/unit/operations/get_more.test.ts

```diff
@@ -192,6 +192,54 @@ describe('GetMoreOperation', function () {
         }
       });
     });
+
+    context('error cases', () => {
```

Comment:
what about if server is undefined, or was that code always unreachable?

Reply:
Should always be unreachable, server selection would have to fail, and we would have something wrong in our error handling of server selection if we proceeded to execute with a nullish server.